src/streams/stream.interface.ts

```diff
@@ -1,17 +1,75 @@
 import {SerializableClassInterface} from '../serializable/serializable.class.interface';
 import {StreamJSONInterface} from './stream';
+
+/**
+ * A stream consists of an array of data like this for example
+ * AltitudeStream: [100, 110, null, 112, 200, null, 300]
+ * The sampling rate of streams is fixed to 1s
+ * That means that on the above example the duration is seconds would be:
+ * [1,2,3,4,5,6,6]
+ *
+ * null means no value and that means missing data.
+ */
 export interface StreamInterface extends SerializableClassInterface {
+
+  /**
+   * Type of the string. Can be any of the datatypes stored in the dataStore.
+   * @todo make this an 'enum'
```

Comment:
Nope :-) Keeping todo's there for reminders when I have free time etc. 

For example the type here (that needs to be an enum (perhaps) needs quite some work, and was there I think since the start of this class.

Reply:
Ok this is what i though, just to be sure before clicking the button :)